docs/usage-guide.md

```diff
@@ -191,6 +191,34 @@ Chroma uses collection names in the url, so there are a few restrictions on nami
 - The name must not contain two consecutive dots.
 - The name must not be a valid IP address.
 
+## HuggingFace Embeddings
```

Comment:
I think we'd prefer to keep this section clean, without bringing up HF embeddings. Let's revert the change to this file, but keep the others.